laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -394,130 +342,53 @@ public function selectMilestones($id)
-            $attending2 = substr($attending_string, 0, strpos($attending_string, "["));
-
-            // Store second choice data
-            $data2["schedule"] = $schedule_data2[0];
-            $data2["attending"] = $attending2;
-            // Find previous milestone and educational objective
-            $option2 = Option::where("date", $date)
-                ->where("resident", $resident)
-                ->where("option", 2)
-                ->get();
-            if (sizeof($option2) > 0) {
-                $milestone2 = Milestone::where("id", $option2[0]["milestones"])->get();
-                if (sizeof($milestone2) > 0) {
-                    $data2["milestone"] = $milestone2[0];
-                }
-                $data2["objective"] = $option2[0]["objectives"];
-                $data2["pref_anest"] = $option2[0]["anesthesiologist_id"];
-            }
-        }
-
-        //Initiate second choice data
-        $data3 = [
-            "schedule" => null,
-            "attending" => null,
-            "milestone" => null,
-            "objective" => null,
-            "pref_anest" => null,
-        ];
-        $choice++;
-        // If the third choice exists, get information for third choice
-        if ($split[2] != 0) {
-            $schedule_data3 = ScheduleData::where("id", $split[2])->get();
-            $attending_string = $schedule_data3[0]["lead_surgeon"];
-            $attending = substr(
-                $attending_string,
-                strpos($attending_string, "[") + 1,
-                strpos($attending_string, "]") - (strpos($attending_string, "[") + 1)
-            );
-            $attending3 = substr($attending_string, 0, strpos($attending_string, "["));
-
-            // Store second choice data
-            $data3["schedule"] = $schedule_data3[0];
-            $data3["attending"] = $attending3;
-            // Find previous milestone and educational objective
-            $option3 = Option::where("date", $date)
-                ->where("resident", $resident)
-                ->where("option", 3)
-                ->get();
-            if (sizeof($option3) > 0) {
-                $milestone3 = Milestone::where("id", $option3[0]["milestones"])->get();
-                if (sizeof($milestone3) > 0) {
-                    $data3["milestone"] = $milestone3[0];
+        for ($i = 0; $i < 3; $i++) {
+            $resident_data[$i] = [
+                "schedule" => null,
+                "attending" => null,
+                "milestone" => null,
+                "objective" => null,
+                "pref_anest" => null,
+            ];
+            if ($split[$i]) {
+                $choice = $i + 1;
+                $schedule_data[$i] = ScheduleData::where("id", $split[$i])->get();
+                $attending_string = $schedule_data[$i][0]["lead_surgeon"];
+                preg_match("/(.+) \[(\d+)\]/", $attending_string, $matches); // get name of the lead surgeon
+                $attending = $matches[1];
```

Comment:
Should `attending` be renamed to `lead_surgeon`? I feel like naming throughout REMODEL can get confusing when the attending sometimes refers to anesthesiologists and other times lead_surgeons

Reply:
Yes! This is such a confusing issue throughout the code (likely b/c the people who picked the variable names aren't doctors and therefore, may not have understood the distinction).

One of the things I think future teams might benefit a lot from is including a "Weird Stuff To Be Aware Of" section in our transition doc. imo, this 100% belongs there.

Reply:
Just changed a few variable names to make it a little more clear, should be out next push